packages/tooling/fast-tooling/src/data-utilities/mapping.mdn-data.ts

```diff
@@ -523,24 +528,54 @@ export function resolveCSSPropertySyntax(
     };
 }
 
+function conformsToOptions(
+    cssPropertyMapOptions: CSSPropertyMapOptions,
+    key: string,
+    value: MDNCSSPropertyConfig
+): boolean {
+    let validatesAgainstOptions: boolean = true;
+
+    if (
+        cssPropertyMapOptions.status !== undefined &&
+        cssPropertyMapOptions.status !== value.status
+    ) {
+        validatesAgainstOptions = false;
+    }
+
+    return validatesAgainstOptions;
+}
+
 /**
  * Maps MDN CSS properties to a CSS properties dictionary
  * for tooling consumption
  */
-export function mapCSSProperties(mdnCSS: MDNCSS): CSSPropertiesDictionary {
+export function mapCSSProperties(
+    mdnCSS: MDNCSS,
+    cssPropertyMapOptions: CSSPropertyMapOptions = {}
+): CSSPropertiesDictionary {
     const syntaxKeys: string[] = Object.keys(mdnCSS.syntaxes);
     const typeKeys: string[] = Object.keys(mdnCSS.types);
 
     return Object.entries(mdnCSS.properties)
         .map(
             ([key, value]: [string, MDNCSSPropertyConfig]): CSSProperty => {
-                return {
-                    name: key,
-                    appliesTo: value.appliesto,
-                    syntax: resolveCSSPropertySyntax(value, key, syntaxKeys, typeKeys),
-                };
+                if (conformsToOptions(cssPropertyMapOptions, key, value)) {
+                    return {
+                        name: key,
+                        appliesTo: value.appliesto,
+                        syntax: resolveCSSPropertySyntax(
+                            value,
```

Comment:
Is perf a significant consideration here? If so I would recommend combining this all into the reduce function. As-is there are three iterations over the array where there could one.

Reply:
Not really a consideration as it's run ad-hoc, but no reason to not make the change :)